azure-pipelines.yml

```diff
@@ -15,32 +15,70 @@ pr:
 
 
 jobs:
-- job: Windows
+- job: Azure_Tests
```

Comment:
I have a feeling I'd mess it up if I try to help merge this - but I'm happy to give it a go if you're short on time for this @tylerjereddy